bindata/network/openshift-sdn/controller.yaml

```diff
@@ -54,6 +54,59 @@ spec:
         volumeMounts:
         - mountPath: /env
           name: env-overrides
+      - name: kube-rbac-proxy
+        image: {{.KubeRBACProxyImage}}
+        command:
+          - /bin/bash
+          - -c
+          - |
+            #!/bin/bash
+            set -euo pipefail
+            TLS_PK=/etc/pki/tls/metrics-certs/tls.key
+            TLS_CERT=/etc/pki/tls/metrics-certs/tls.crt
+
+            # As the secret mount is optional we must wait for the files to be present.
+            # The service is created in monitor.yaml and this is created in controller.yaml.
+            # If it isn't created there is probably an issue so we want to crashloop.
+            TS=$(date +%s)
+            WARN_TS=$(( ${TS} + $(( 20 * 60)) ))
+            HAS_LOGGED_INFO=0
+
+            log_missing_certs(){
+                CUR_TS=$(date +%s)
+                if [[ "${CUR_TS}" -gt "WARN_TS"  ]]; then
```

Comment:
Here you're probably missing a dollar sign in front of `WARN_TS`, right? :)

Reply:
Yep. Thank you!